### PR TITLE
Add initial revision related tests

### PR DESCRIPTION
**Pull request by uvindra**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: import-export-cli/integration/testutils/api_testUtils.go [hunk 1]
```diff
@@ -274,6 +317,23 @@ func exportAPI(t *testing.T, name string, version string, provider string, env s
 	return output, err
 }
 
+func exportAPIRevision(t *testing.T, name string, version string, provider string, revision string, env string) (string, error) {
```
Comment (wasuradananjith) [inline import-export-cli/integration/testutils/api_testUtils.go:320]:
```suggestion
func exportAPIRevision(t *testing.T, name, version, provider, revision, env string) (string, error) {
```
Shall we change like this to improve the readability and resuce the characters in a code line?

File: import-export-cli/integration/testutils/api_testUtils.go [hunk 2]
```diff
@@ -455,33 +515,65 @@ func ValidateExportedAPIStructure(t *testing.T, args *ApiImportExportTestArgs) {
...
 	// Read the api.yaml file in the exported directory
-	fileData, err := ioutil.ReadFile(relativePath + string(os.PathSeparator) + api.Name + "-" + api.Version + APIYamlFilePath)
+	fileData, err := ioutil.ReadFile(filepath.Join(unzipedProjectPath, APIYamlFilePath))
+
 	if err != nil {
 		t.Error(err)
 	}
+
+	validateAPIStructure(t, &fileData, sampleFile)
+	//validateAPIValues(t, &fileData, api)
```
Comment (wasuradananjith) [inline import-export-cli/integration/testutils/api_testUtils.go:557]: Any reason to have this commented code segment? Shall we remove it?

Comment (uvindra) [reply import-export-cli/integration/testutils/api_testUtils.go:557]: Currently this is breaking when we try to compare value from REST API vs what is there in project. Need to do changes to the DTO at test level to make this possible. Currently this requires extensive changes so did not include this in current PR. 

I'll remove this for now, maybe I can come up with a better way of doing this

Comment (uvindra) [reply import-export-cli/integration/testutils/api_testUtils.go]: Fixed with https://github.com/wso2/product-apim-tooling/pull/728/commits/0901d9dcfc9501ea1bac6682987f76eae977e297

File: import-export-cli/integration/testutils/api_testUtils.go (hunk 1 above)
Comment (uvindra) [reply import-export-cli/integration/testutils/api_testUtils.go]: Fixed with https://github.com/wso2/product-apim-tooling/pull/728/commits/0901d9dcfc9501ea1bac6682987f76eae977e297

File: import-export-cli/integration/testutils/api_testUtils.go (hunk 2 above)
Comment (wasuradananjith) [reply import-export-cli/integration/testutils/api_testUtils.go]: Sure. Thanks

**Review by wasuradananjith — APPROVED** [review]

**Review by Chamindu36 — APPROVED** [review]

**Pull request merged**